internal/lookoutv2/repository/getjobrunerror.go

```diff
@@ -29,24 +29,16 @@ func NewSqlGetJobRunErrorRepository(db *pgxpool.Pool, decompressor compress.Deco
 
 func (r *SqlGetJobRunErrorRepository) GetJobRunError(ctx context.Context, runId string) (string, error) {
 	var rawBytes []byte
-	err := r.db.BeginTxFunc(ctx, pgx.TxOptions{
+	err := pgx.BeginTxFunc(ctx, r.db, pgx.TxOptions{
 		IsoLevel:       pgx.RepeatableRead,
 		AccessMode:     pgx.ReadOnly,
 		DeferrableMode: pgx.Deferrable,
 	}, func(tx pgx.Tx) error {
-		rows, err := tx.Query(ctx, "SELECT error FROM job_run WHERE run_id = $1 AND error IS NOT NULL", runId)
```

Comment:
I noticed that the code here, and other parts, used tx.Query just to query a single row and found it confusing. switched to tx.QueryRow instead.

Reply:
looks good- as an aside (and I know you didn't write the original query here) I don't think we need a transaction function here.  We only execute a single sql statement so we can probably just do `r.db.QueryRows()`.  Maybe one for a subsequent PR.